io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --filesystem=home:rw
+modules:
+  - name: obsidian
+    buildsystem: cmake
```

Comment:
it doesn't build with cmake-ninja? It's prefered.

Reply:
Why is it preferred?

Reply:
because it's faster with ninja.

Reply:
Updated to use cmake-ninja